src/api/app/controllers/webui/staging_workflows/request_ignorations_controller.rb

```diff
@@ -0,0 +1,60 @@
+module Webui
+  module StagingWorkflows
+    class RequestIgnorationsController < WebuiController
+      layout 'webui2/webui'
+
+      before_action :require_login, except: [:index]
+      before_action :set_bootstrap_views
+      before_action :set_staging_workflow
+      before_action :set_request_ignoration, only: [:destroy]
+      after_action :verify_authorized, except: [:index]
+
+      def index
+        @request_ignorations = @staging_workflow.request_ignorations
+      end
+
+      def create
+        authorize @staging_workflow
+
+        request_ignoration = @staging_workflow.request_ignorations.build(request_ignoration_params)
+
+        if request_ignoration.save
+          flash[:success] = "Request #{request_ignoration.bs_request_id} was successfully ignored"
```

Comment:
@bgeuken:

> As pointed out by [coolo](https://github.com/openSUSE/open-build-service/pull/6201#discussion_r232258521) we should show the number here. Since that's how users can identify requests in OBS

It is already showing the number. `bs_request` is defined in the model with this line:

```
belongs_to :bs_request, primary_key: :number
```

Reply:
@coolo 

> you trust your future yourself to realize the bs_request_id is the requests's number

I don't think it is something you can change that easily and it is something conceptual. We don't look the BsRequest by id, but by number (I even wonder why we have an extra `id` column`). I think we should leave it as it is now.

Reply:
> It is already showing the number. bs_request is defined in the model with this line:

And why? The id is the primary key so I would suggest to use it. This adds just more complexity to the assocations.

> I even wonder why we have an extra id column

This is historical, when you have several MySQL instances, the id gets incremented in steps (instance one increments the by even numbers and instance two by uneven). This would leave some ids unused. As the bs request is customer facing, maintenance requested that there are no gaps between the numbers. This is the reason why there is an id and number. Might not be the best solution but as said, historical ...

Reply:
Really, the number is basically the name of the request - while the id is what we use for database relations. So please use the id for that, but show the number. If it's easier, pick id *and* number into your model.

Reply:
@coolo yep, that's sums it up nice.

Reply:
`, primary_key: :number` was removed from `belongs_to :bs_request`. So now `id` is used now as primary key.